From a2bee043f1410bcd2d2e405a545954118ef38f1e Mon Sep 17 00:00:00 2001
From: Aswin Murugan <quic_aswinm@quicinc.com>
Date: Fri, 18 Apr 2025 10:54:33 +0530
Subject: [PATCH] PENDING: Removed preboot default env variable

Removed preboot condition as scsi scan, since scsi_scan is internally handled as part of
scsi_get_blk(), it has been removed

Signed-off-by: Aswin Murugan <quic_aswinm@quicinc.com>
Upstream-Status: Pending
---
 board/qualcomm/default.env | 1 -
 1 file changed, 1 deletion(-)

diff --git a/board/qualcomm/default.env b/board/qualcomm/default.env
index 1101ecab67..98291979fd 100644
--- a/board/qualcomm/default.env
+++ b/board/qualcomm/default.env
@@ -1,7 +1,6 @@
 stdin=serial,button-kbd
 stdout=serial,vidconsole
 stderr=serial,vidconsole
-preboot=scsi scan
 bootargs=root=/dev/disk/by-partlabel/system rw rootwait console=ttyMSM0,115200n8 earlycon
 fastboot=fastboot -l $fastboot_addr_r usb 0
 do_boot=bootefi bootmgr
-- 
2.34.1

